documentation-site/components/yard/config/fixed-marker.ts

```diff
@@ -76,6 +88,72 @@ export const fixedMarkerProps = {
     type: PropTypes.String,
     description: 'Color to render for content.',
   },
+  badgeEnhancerSize: {
```

Comment:
I don't know.. in my experience hiding the options decreases the chances people will discover them. I personally think the number or options is decent, not too long yet: 
<img width="605" alt="image" src="https://user-images.githubusercontent.com/834796/152881953-1d0a98eb-cedd-4158-ba1d-c2a7d7a0989a.png">

The only two that I think will be used much less frequently are: `labelEnhancerColor` and `labelEnhancerStrokeColor`.  But not sure if it makes sense to hide them. That said I'm open to other opinions.

Reply:
Yeah I like the amount there is

Reply:
A lot of these props are usually implemented as overrides in baseweb (colors), so seeing them here seems like we're mixing functional/constant based options (size/needle type/enhancers/label) with style choices. 

On another note, it there a reason we can't implement these options as overrides so this component follows the pattern established in the rest of the library?

Reply:
The reason I originally implemented props for `color` and `backgroundColor` is because the design spec states that the sub components must be the same color (eg: the needle and pinhead cannot be different colors). I thought the easiest way to do this was with a single prop for color and a single prop for background color, mostly because the style overrides seem to focus on particular elements? 

So in order to make the fixed marker red, the developer would need to pass two different color overrides for Needle and Pinhead? Or is there a way we can use something like a `Base` override that can serve as a single single override for every subelement that needs to be changed?

Reply:
Yea, I can see how `badgeEnhancerColor`, `badgeEnhancerBackground` and `labelEnhancerColor`, `labelEnhancerStrokeColor` can easily be removed and configured via overrides.

`color` and `background` are a little bit trickier because of the synchronization @dbabbs mentioned above. Is there a good solution for that?

Reply:
Ok, I've got an idea that satisfies all of our requirements.
We can remove the `color` and `background` props in favor of a single `kind` prop.

This is similar to what the [Notification](https://baseweb.design/components/notification/) component does.

For Fixed marker we can implement 3 different `KIND`s. The `kind` prop will control the color on both the pinhead, needle, and drag shadow.

- `KIND.default`: black
- `KIND.accent`: blue
- `KIND.negative`: red
These 3 colors are the majority of the fixed markers used within Uber. If the developer needs to use different colors, they can use the overrides.